### PR TITLE
Update prebid to v7.26.0

### PR DESCRIPTION
**Pull request by Jakeii**
## What does this change?
Update prebid to v7.26.0

See: https://github.com/guardian/Prebid.js/pull/130

## Does this change need to be reproduced in dotcom-rendering ?

- [x] No
- [ ] Yes (please indicate your plans for DCR Implementation)

### Does this affect other platforms?

- [ ] AMP 
- [ ] Apps
- [ ] Other (please specify)

### Does this affect GLabs Paid Content Pages? Should it have support for Paid Content?

- [ ] No
- [ ] Yes (please give details)

### Does this change break ad-free?

- [ ] No
- [ ] It did, but tests caught it and I fixed it
- [ ] It did, but there was no test coverage so I added that then fixed it

### Does this change update the version of CAPI we're using?

- [ ] No, all the existing database files are just fine
- [ ] Yes, and I have [re-run all the tests locally and checked in all the updated data/database/xyz files](https://github.com/guardian/frontend/blob/main/docs/03-dev-howtos/15-updating-test-database.md)

### Accessibility test checklist

- [ ] [Tested with screen reader](https://accessibility.gutools.co.uk/testing/web/screen-readers/)
- [ ] [Navigable with keyboard](https://accessibility.gutools.co.uk/testing/web/keyboard-navigation/)
- [ ] [Colour contrast passed](https://accessibility.gutools.co.uk/testing/web/colour-contrast/)

### Tested

- [x] Locally
- [x] On CODE (optional)

### CODE REVIEW

File: static/src/javascripts/projects/commercial/modules/header-bidding/prebid/prebid.spec.ts [hunk 1]
```diff
@@ -105,13 +106,23 @@ describe('initialise', () => {
 			maxNestedIframes: 10,
 			mediaTypePriceGranularity: {},
 			priceGranularity: 'custom',
-			publisherDomain: 'http://testurl.theguardian.com',
+			publisherDomain: null,
```
Comment (Jakeii) [inline static/src/javascripts/projects/commercial/modules/header-bidding/prebid/prebid.spec.ts:109]: this was deprecated in v7 but is still null for some reason

File: package.json [hunk 2]
```diff
@@ -41,7 +41,7 @@
 		"object-fit-videos": "^1.0.3",
 		"ophan-tracker-js": "1.4.0",
 		"preact": "^10.5.13",
-		"prebid.js": "https://github.com/guardian/Prebid.js#0e9f87a",
+		"prebid.js": "guardian/Prebid.js#7.26.0",
```
[the remark this answers is not in the document]
Comment (Jakeii) [reply package.json:44]: Thats the same behaviour as before though isn't it? before it was fixed to `0e9f87a`, now it's fixed to `7.26.0` which is effectively an alias of `8532210`.

If we need to make further changes outside of the prebid.js versions, we can create a subsequent release/tag of `7.26.0-patch.1` or something.

[the remark this answers is not in the document]
Comment (arelra) [reply package.json:44]: I feel like using commits are easier to point to? If we use tags then we will need to remember to generate tags for each commit to our fork. Would be good to hear other opinions.

**Review by chrislomaxjones — APPROVED** [review]
Looks good to me!

**Comment by Jakeii** [comment]
Cool happy to go with consensus RE the prebid.js version!

**Pull request merged**